patches/src/main/kotlin/app/revanced/patches/pandora/skips/UnlimitedSkipsHookPatch.kt

```diff
@@ -0,0 +1,9 @@
+package app.revanced.patches.pandora.skips
+
+import app.revanced.patches.pandora.misc.hook.hookPatch
+
+@Suppress("unused")
+val unlimitedSkipsHookPatch = hookPatch(
+    name = "Unlimited skips",
+    hookClassDescriptor = "Lapp/revanced/extension/pandora/skips/UnlimitedSkipsHook;",
+)
```

Comment:
Make sure to also rename the file
```suggestion
val enableUnlimitedSkipsHookPatch = hookPatch(
    name = "Enable unlimited skips",
    hookClassDescriptor = "Lapp/revanced/extension/pandora/skips/UnlimitedSkipsHook;",
)
```